### PR TITLE
redirects update

### PR DESCRIPTION
**Pull request by divyaamunugama**
$subject

### CODE REVIEW

File: en/mkdocs.yml [hunk 1]
```diff
@@ -786,289 +786,289 @@ plugins:
...
-              'learn/configuring-user-name-recovery.md': 'https://is.docs.wso2.com/en/6.0.0/'
-              'learn/configuring-account-locking.md': 'https://is.docs.wso2.com/en/6.0.0/'
-              'learn/configuring-admin-initiated-account-locking.md': 'https://is.docs.wso2.com/en/6.0.0/'
-              'learn/configuring-account-locking-due-to-failed-login-attempts.md': 'https://is.docs.wso2.com/en/6.0.0/'
-              'learn/configuring-account-disabling.md': 'https://is.docs.wso2.com/en/6.0.0/'
-              'learn/pending-account-status.md': 'https://is.docs.wso2.com/en/6.0.0/'
-              'learn/configuring-single-sign-on-saml.md': 'https://is.docs.wso2.com/en/6.0.0/'
-              'learn/configuring-single-sign-on-oidc.md': 'https://is.docs.wso2.com/en/6.0.0/'
+              'learn/tutorials.md': 'https://is.docs.wso2.com/en/6.0.0/guides/before-you-start/'
+              'learn/adding-users-and-roles.md': 'https://https://is.docs.wso2.com/en/6.0.0/guides/identity-lifecycles/user-management/'
```
Comment (nilminiwso2) [inline en/mkdocs.yml:790]: this one is wrong isn't it
`https://` duplicated

Comment (divyaamunugama) [reply en/mkdocs.yml:790]: Yeas, Will update it thankss

Comment (divyaamunugama) [inline en/mkdocs.yml:790]:
```suggestion
              'learn/adding-users-and-roles.md': 'https://is.docs.wso2.com/en/6.0.0/guides/identity-lifecycles/user-management/'
```

File: en/mkdocs.yml [hunk 2]
```diff
@@ -1150,7 +1150,7 @@ plugins:
               'learn/xacml-sample-for-an-online-trading-application.md': 'https://is.docs.wso2.com/en/6.0.0/'
               'learn/log-in-to-the-identity-server-using-another-identity-server-saml2.md': 'https://is.docs.wso2.com/en/6.0.0/'
               'learn/login-to-identity-server-using-another-identity-server-oauth2.md': 'https://is.docs.wso2.com/en/6.0.0/'
-              'learn/cors.md': 'https://is.docs.wso2.com/en/6.0.0/'
+              'learn/cors.md': 'deploy/configure-cors/'
```
Comment (nilminiwso2) [inline en/mkdocs.yml:1153]: what happened here?

File: en/mkdocs.yml [hunk 3]
```diff
@@ -786,335 +786,335 @@ plugins:
...
-              'learn/configuring-user-name-recovery.md': 'https://is.docs.wso2.com/en/6.0.0/'
-              'learn/configuring-account-locking.md': 'https://is.docs.wso2.com/en/6.0.0/'
-              'learn/configuring-admin-initiated-account-locking.md': 'https://is.docs.wso2.com/en/6.0.0/'
-              'learn/configuring-account-locking-due-to-failed-login-attempts.md': 'https://is.docs.wso2.com/en/6.0.0/'
-              'learn/configuring-account-disabling.md': 'https://is.docs.wso2.com/en/6.0.0/'
-              'learn/pending-account-status.md': 'https://is.docs.wso2.com/en/6.0.0/'
-              'learn/configuring-single-sign-on-saml.md': 'https://is.docs.wso2.com/en/6.0.0/'
-              'learn/configuring-single-sign-on-oidc.md': 'https://is.docs.wso2.com/en/6.0.0/'
+              'learn/tutorials.md': 'https://is.docs.wso2.com/en/6.0.0/guides/before-you-start/'
+              'learn/adding-users-and-roles.md': 'https://https://is.docs.wso2.com/en/6.0.0/guides/identity-lifecycles/user-management/'
```
Comment (nilminiwso2) [inline en/mkdocs.yml:790]:
```suggestion
              'learn/adding-users-and-roles.md': 'https://is.docs.wso2.com/en/6.0.0/guides/identity-lifecycles/user-management/'
```

**Review by nilminiwso2 — APPROVED** [review]

**Pull request merged**